web/beacon-app/src/features/auth/routes/Registration.tsx

```diff
@@ -0,0 +1,74 @@
+import { Toast } from '@rotational/beacon-core';
+import { FormikHelpers } from 'formik';
+import { Link, useNavigate } from 'react-router-dom';
+
+import slugify from '@/utils/slugifyDomain';
+
+import RegistrationForm from '../components/RegistrationForm';
+import { useRegister } from '../hooks/useRegister';
+import { NewUserAccount } from '../types/RegisterService';
+
+export function Registration() {
+  const register = useRegister();
+  const navigateTo = useNavigate();
+  const handleSubmitRegistration = (
+    values: NewUserAccount,
+    helpers: FormikHelpers<NewUserAccount>
+  ) => {
+    values.domain = slugify(values.domain);
+
+    register.createNewAccount(values, {
+      onSuccess: (_response) => {
+        navigateTo('/auth/verify-account', { replace: true });
+      },
+      onSettled: (_response) => {
+        helpers.setSubmitting(false);
+      },
+    });
+  };
+
+  return (
+    <>
+      <Toast
+        isOpen={register.hasAccountFailed}
+        title="Something went wrong, please try again later."
+        description={(register.error as any)?.response?.data?.error}
+      />
+      <div className="flex flex-col gap-4 px-4 py-8 text-sm sm:p-8 md:flex-row md:p-16 xl:text-base">
+        <div className="space-y-4 rounded-md border border-[#1D65A6] bg-[#1D65A6] p-4 text-white sm:p-8 md:w-2/6">
+          <h1 className="text-center font-bold">
+            Building event-driven applications can be fast, convenient and even fast!
```

Comment:
@elysee15 @masskoder  just noticed a typo here - the text should be:

Building event-driven applications can be fast, convenient and even fun!